charts/azure/azure-disk-loss/azure-disk-loss.chartserviceversion.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: litmuchaos.io/v1alpha1
+kind: ChartServiceVersion
+metadata:
+  name: azure-disk-loss 
+  version: 0.1.0
+  annotations:
+    categories: Kubernetes
+    vendor: Chaos Native
+    support: https://app.slack.com/client/T09NY5SBT/CNXNB0ZTN
+spec:
+  displayName: azure-disk-loss 
+  categoryDescription: >
+    
+  keywords: 
+    - Kubernetes
+    - Azure
+    - Termination
+  platforms: 
+    - Azure
+  maturity: alpha
+  maintainers: 
+    - name: avaakash 
+      email: akash@chaosnative.com 
+  minKubeVersion: 1.12.0 
+  provider: 
+    name: ChaosNative
+  labels:
+    app.kubernetes.io/component: chartserviceversion
+    app.kubernetes.io/version: latest
+  links: 
+    - name: Source Code
+      url: https://github.com/litmuschaos/litmus-go/tree/master/experiments/azure/disk-loss/experiment
+    - name: Documentation 
+      url: https://litmuschaos.github.io/litmus/experiments/categories/azure/azure-disk-loss/
```

Comment:
can you please check the correct docs link.

Reply:
The docs PR isn't merged yet that why it's showing 404.